src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -141,6 +159,21 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
 
   const [isRadioPlaying, setIsRadioPlaying] = useState(false);
   const { volume, setVolume } = useRadio(isRadioPlaying, sound);
+  const handleMute = useCallback(
+    (volume: number) => (volume !== 0 ? 0 : 100),
```

Comment:
Should volume reset to full blast or should it take previous (pre-mute) value from some place else like browser API or global redux state?

Reply:
fixed 61d9df7, replaced 100 with volume

Reply:
@sunny-viktoryia This was changed back and forth. 

First `100` -> `volume` which caused the function to always return `0` due to scope shadowing.

Then, because of lack of volume controls (even now with https://github.com/sparkletown/sparkle/pull/2028 Animatemap only supports mute/unmute)  the `volume` -> `100` was reverted in another comment I noticed in a previous commit

This way at least it doesn't cause a bug.